src/common/channel/channel.test.ts

```diff
@@ -55,6 +56,10 @@ describe("channel", () => {
       );
 
       await applicationBuilder.render();
+
+      const closeAllWindows = mainDi.inject(closeAllWindowsInjectable);
+
+      closeAllWindows();
```

Comment:
I think this implies that something else is broken.

Reply:
At the moment `applicationBuilder` doesn't separate the process of `startMainApplication` from `startWindow` and both of them happen when you call `applicationBuilder.render`. This means that in some cases we wan't to test what happens if we do something in `main` before we start the `window`. 

However, currently these cases are rare and therefore we didn't separate those processes yet. This is TBD later.